index.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
     'mocha/no-pending-tests': 'error',
     'mocha/no-skipped-tests': 'warn',
     'mocha/valid-test-description': 'error',
+    'no-template-curly-in-string': 'off',
```

Comment:
This PR suggest turning off a rule to accommodate syntax for which this rule was never meant to apply.  This rule - http://eslint.org/docs/rules/no-template-curly-in-string - is a valid rule and one we should continue to leverage in our codebases.  If an application needs to make an exception to this rule than it should be done at the application-level where the tradeoffs are understood before being entered into and the outcome of the decision is limited to only the application.  Even if every application needs this (as you previously somewhat suggested @EWhite613) I believe the case can be made that the value of this rule, enforced in its original content, outweighs having to turn it off in each instance where an application does not want it in order to use non-conforming Bunsen syntax.

Speaking of exceptions, in which file format are these strings being encountered?  Are they in JS files where `// eslint-disable-next-line no-template-curly-in-string` can be employed?